pkg/asset/machines/worker.go

```diff
@@ -28,6 +28,10 @@ import (
 func defaultAWSMachinePoolPlatform() awstypes.MachinePool {
 	return awstypes.MachinePool{
 		InstanceType: "m4.large",
+		EC2RootVolume: awstypes.EC2RootVolume{
+			Type: "gp2",
+			Size: 32,
```

Comment:
`defaultAWSMachinePoolPlatform` is currently used for both masters and workers, but this is way too small for masters (because AWS ties IOPS to size).  See bca6a92f (#737) for the last master size bump (and source of the still-current 120 GiB master size).

See [here][1] for one pattern to follow for separating worker and master defaults (setting them just after `defaultAWSMachinePoolPlatform()`), although I'd personally be fine dropping `defaultAWSMachinePoolPlatform` entirely.

[1]: https://github.com/openshift/installer/pull/1069#issuecomment-454675830

Reply:
> See here for one pattern to follow for separating worker and master defaults (setting them just after defaultAWSMachinePoolPlatform()), although I'd personally be fine dropping defaultAWSMachinePoolPlatform entirely.

hmm, not sure if it is easy to repeat instance type or just override the default when needed ;)